tarpc/src/serde_transport.rs

```diff
@@ -669,4 +713,25 @@ mod tests {
         assert_matches!(transport.next().await, None);
         Ok(())
     }
+
+    #[cfg(all(unix, feature = "unix"))]
+    #[tokio::test]
+    async fn uds_on_existing_transport() -> io::Result<()> {
+        use super::unix;
+        use super::*;
```

Comment:
Just curious, what else was needed besides `super::unix`?

Reply:
Ah, I think the imports on line 565 just need to be updated to include StreamExt and SinkExt. I see another test already imports super::*, so I'll just update this in a separate commit.